### ISSUE TITLE
Části zakázek

### ISSUE THREAD

**Issue by vrana** [HlidacStatu/verejne-zakazky#12]
Výsledná struktura moc nepočítá se zakázkami rozdělenými na části ([ukázka](https://zakazky.lesycr.cz/contract_display_9026.html)). Pole `Dodavatele` dovoluje uložit více dodavatelů, tam se aspoň dají uložit dodavatelé jednotlivých částí. Ale není kam uložit název, popis a předpokládanou hodnotu jednotlivých částí.

Dokumenty vztahující se k částem jsou v JSON exportu uloženy u hlavní zakázky.

**Comment by michalblaha** [comment]
Ano, jedna se o zjednoduseni. Doplnim to ale do finalni dat. struktury
